extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -285,12 +288,45 @@ export class ProjectsController {
 		}
 	}
 
+	/**
+	 * Publishes a project to a new Azure server
+	 * @param context a treeItem in a project's hierarchy, to be used to obtain a Project or the Project itself
+	 * @param deployProfile deploy profile
+	 */
+	public async publishToAzure(context: Project | dataworkspace.WorkspaceTreeItem, deployProfile: ISqlDbDeployProfile): Promise<void> {
```

Comment:
it would be nice for the function name to also mention that it publishes to a new Azure server so there isn't any confusion if this could also publish to an existing Azure server